src/mmw/js/src/app.js

```diff
@@ -50,6 +50,19 @@ var App = new Marionette.Application({
         // new project.
         this.currentProject = null;
 
+        $(window)
+            .on('shutterbug-saycheese', function() {
+                // Set fixed width before screenshot to constrain width to viewport
+                $('#model-output-wrapper, .map-container').css({
+                    'width': window.innerWidth
+                });
+            })
+            .on('shutterbug-asyouwere', function() {
+                // Reset after screenshot has been taken
+                $('#model-output-wrapper, .map-container').css({
+                    'width': ''
+                });
+            });
```

Comment:
Consider moving this code (and the following related `shutterbug` line`) to a helper function that can be called from `initialize` just for the sake of making the length of the function shorter.